packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -22,6 +22,7 @@
   $input-type: null
 ) {
   $input-text-color: get-color-token-from-bg($bg-color);
+  $input-text-color-disabled: get-color-token-from-bg($bg-color, "disabled", "gray-50");
```

Comment:
Compares theme `disabled` color against form background colors. 

Falls back to standard `gray-50` if `disabled` doesn't match color contrast. The goal here is to provide the standard disabled color if the user-updated color does not pass color contrast. 

This seemed to me like a better solution than defaulting to `ink` which doesn't "read" as disabled.

A warning is provided if both colors fail color contrast checks.

### Dark backgrounds

Changing the form background color to a dark color will fail a color contrast check, leaving low contrast between disabled content and its background BUT this matches the current fallback behavior of the color "ink" which is the default fallback for most components.

Are we ok relying on the compilation warning in these situations?